tft.py

```diff
@@ -788,7 +802,8 @@ def main():
 
     setup_hotkeys()
 
-    LCU_INTEGRATION.connect_to_lcu()
+    if not LCU_INTEGRATION.connect_to_lcu():
+        sys.exit(1)

```

Comment:
Hmm, does it make sense to fully abort here, or should it instead restart the League client in case it was launched when it was closed? That's my main "launch method" these days when running the bot 😛

Reply:
In my ideal world, we exit here since we want the client to be available on the first launch since that's where we can grab the most reliable league installation path. But if you prefer to estimate it with the previous method and try to start the league client here, I can add that :)

Reply:
Ah gotcha! I think it would be useful to include, so maybe an 'is client running' check first, if so, update install path from there, otherwise, attempt the existing method?